### PR TITLE
require detox device to solve ReferenceError

### PR DESCRIPTION
**Pull request by yaron1m**
I was trying to run detox sample test on my app, which is using:
- Expo
- Typescript
- Jest

And I always get the following error:
```
    ReferenceError: device is not defined

      at Object.reloadApp (../node_modules/detox-expo-helpers/index.js:68:3)
```

When I added the following require:
`const { device } = require('detox');`
The issue was solved.

In order to do so, I had to add detox as a dependency.

### CODE REVIEW

**Comment by chakrihacker** [comment]
Please merge this

File: package.json [hunk 1]
```diff
@@ -6,7 +6,8 @@
   "devDependencies": {},
   "dependencies": {
     "child-process-promise": "^2.2.1",
-    "xdl": "^51.5.0",
-    "semver": "^5.6.0"
+    "detox": "^12.2.0",
```
Comment (chakrihacker) [inline package.json]: Maybe you can add detox as peer dependency

Comment (yaron1m) [reply package.json]: Done

**Comment by chakrihacker** [comment]
Hi, I tried adding device import, but I am encountering launchApp is null issue when I added it.

**Comment by yaron1m** [comment]
Did it work when detox was a dependency?

**Comment by chakrihacker** [comment]
nope

**Comment by nicoboy11** [comment]
Why this hasn`t been merged?

**Comment by yaron1m** [comment]
I managed to get it all working without this change.
This can be closed
https://github.com/yaron1m/expo-detox-typescript-example

**Pull request closed without merging**